packages/eslint-plugin/src/rules/member-ordering.ts

```diff
@@ -396,8 +396,8 @@ function getNodeType(node: Member): MemberKind | null {
       return node.value && functionExpressions.includes(node.value.type)
         ? 'method'
         : node.readonly
-          ? 'readonly-field'
-          : 'field';
+        ? 'readonly-field'
+        : 'field';
```

Comment:
[Style] Unrelated changes, can you revert?

Reply:
Sure